### ISSUE TITLE
UI issues with new header

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#4668]
Version - master 5/3

Issues:
- [x] Set theme to dark - The three different sections in light theme are very distinguishable, but in dark theme they all pretty much look the same.
![image](https://cloud.githubusercontent.com/assets/11514927/14993331/20d12e04-111f-11e6-9471-01e6a68ca412.png)
![image](https://cloud.githubusercontent.com/assets/11514927/14994218/81e64630-1123-11e6-8594-6a62b3aa60bc.png)

- [x] Environment, Theme & Namespace drop-down selection should be consistent with other menu drop-down.  When you select an item from Catalog, Infrastructure or Admin, the selected item turns blue and stays selected.  In the Environment theme, it turns a opaque color.  Make the sub-selections match the rest of the application.
![image](https://cloud.githubusercontent.com/assets/11514927/14993425/95aa17d6-111f-11e6-841a-61bfb8bd71e9.png)
![image](https://cloud.githubusercontent.com/assets/11514927/14993432/9f7cc178-111f-11e6-9151-4b59283a679d.png)

- [x] Environment & User drop-downs are click while all others are hover - All the regular menu items are hover, but the Environment & User menu (the ones on both ends of the screen) items you have to click to see drop-down.  Make them all hover so they are consistent.

- [x] User Menu is light colored in dark theme
![image](https://cloud.githubusercontent.com/assets/11514927/14993517/f9c4b690-111f-11e6-873b-ea24b08cf90f.png)

- [x] Environment, Namespace & User caret (v) are different colors then regular menu items.  They are light blue while the others are grey.

- [x] User icon & drop-down caret are not centered
![image](https://cloud.githubusercontent.com/assets/11514927/14993615/82c80140-1120-11e6-8fcb-9f750ff89dfe.png)

- [x] Environment, NameSpace & User drop-down menu have space between main menu & submenu.
![image](https://cloud.githubusercontent.com/assets/11514927/14994289/e6775774-1123-11e6-8da6-6b93a6cd558e.png)

**Comment by vincent99** [comment]
- [ ] All the menus should be on hover instead of some (environment, namespace, user) requiring clicks
- [ ] A11y: need to be able to tab through all the menus, use down arrow to open up the dropdowns and select things, etc

**Comment by tfiduccia** [comment]
- [ ] The icons in the submenus are a little too close to the text.  Should match icon/text spacing in environments drop-down
![image](https://cloud.githubusercontent.com/assets/11514927/15060925/406609bc-12e4-11e6-84e3-773577f43ffd.png)

**Comment by tfiduccia** [comment]
Version - master 5.19
Verified fixed
